boards/arm/ip_k66f/doc/index.rst

```diff
@@ -0,0 +1,175 @@
+- `IP-K66F Schematics`_
+
+Supported Features
+==================
+
+The ip_k66f board configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| USB       | on-chip    | USB device                          |
++-----------+------------+-------------------------------------+
+| WATCHDOG  | on-chip    | watchdog                            |
++-----------+------------+-------------------------------------+
+| FLASH     | on-chip    | soc flash                           |
++-----------+------------+-------------------------------------+
+| RTC       | on-chip    | rtc                                 |
++-----------+------------+-------------------------------------+
+
+The default configuration can be found in the defconfig file:
+
+	``boards/arm/ip_k66f/ip_k66f_defconfig``
+
+Micrel/Microchip KSZ8794CNX Ethernet Switch is not currently
+supported.
+
+Connections and IOs
+===================
+
+The K66F SoC has five pairs of pinmux/gpio controllers.
+
++-------+-----------------+---------------------------+
+| Name  | Function        | Usage                     |
++=======+=================+===========================+
+| PTA8  | GPIO            | Red LED                   |
++-------+-----------------+---------------------------+
+| PTA10 | GPIO            | RED LED                   |
++-------+-----------------+---------------------------+
+
+System Clock
+============
+
+The K66F SoC is configured to use the 12 MHz low gain crystal oscillator on the
+board with the on-chip PLL to generate a 180 MHz system clock.
+
+Serial Port
+===========
+
+The K66F SoC has six UARTs. None is used.
+
+USB
+===
+
+The K66F SoC has a USB OTG (USBOTG) controller that supports both
+device and host functions through its micro USB connector (K66F USB).
+Only USB device function is supported in Zephyr at the moment.
+
+Programming and Debugging
+*************************
+
+Build and flash applications as usual (see :ref:`build_an_application` and
+:ref:`application_run` for more details).
+
+Configuring a Debug Probe
+=========================
+
+A debug probe is used for both flashing and debugging the board. This board is
+configured by default to use the :ref:`opensda-jlink-onboard-debug-probe`.
+
+Option 1: :ref:`opensda-jlink-onboard-debug-probe`
```

Comment:
Do we need to mention `Option 1` here, as there is no other options mentioned in this file.

Reply:
Thanks, fixed.